spec/requests/swagger_docs/v5/properties_spec.rb

```diff
@@ -18,13 +18,11 @@
                 required: true,
                 schema: {
                   type: :object,
-                  required: %i[properties],
```

Comment:
I think 'properties' is still required here - otherwise there is no data in the payload?